SMLHelper/Utility/IStorageHelper.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace SMLHelper.V2.Utility
+{
+	interface IStorageHelper
+	{
+		void CacheNewContainer(IItemsContainer container);
+
+		void RecacheContainer(IItemsContainer container);
+
+		void CacheNewHasRoomData(IItemsContainer container, Vector2int itemSize, bool hasRoom);
```

Comment:
```CSharp
void CacheNewContainer(IItemsContainer container);
void RecacheContainer(IItemsContainer container);
void CacheNewHasRoomData(IItemsContainer container, Vector2int itemSize, bool hasRoom);
```
These three methods don't appear to serve a purpose externally.  
If other mods were to call these, we could end up in a very strange state.